flowmachine/flowmachine/features/subscriber/subscriber_tacs.py

```diff
@@ -357,15 +413,70 @@ def _make_query(self):
         return self.joined.get_query()
 
 
-class SubscriberPhoneType(SubscriberFeature):
+class SubscriberHandsetCharacteristic(SubscriberFeature):
     """
-    Class representing which type of phone (Basic, Feature, or Smart) a subscriber
-    is associated with.
+    Class extracting a single characteristic from the handset.
 
     Parameters
-    ----------
+    ---------
```

Comment:
Probably a nitpick, but it's possible that the docs formatter needs the underline to have the same length as the line above it for it to be displayed correctly.
```suggestion
    ----------
```